docs/source/contributing/developing.adoc

```diff
@@ -114,31 +114,31 @@ To install the dependencies, you can run the following command:
 $ make vendor
 ----
 
-This command calls and runs Glide.
-Alternatively, you can run the Glide command directly.
+This command calls and runs Dep.
+Alternatively, you can run the Dep command directly.
 
 ----
-$ glide install -v
+$ dep ensure -v
 ----
 
 [[update-dependencies]]
 === Updating Dependencies
 
-If your work requires a change to the dependencies, you need to update the Glide configuration.
+If your work requires a change to the dependencies, you need to update the Dep configuration.
 
-. Edit *_glide.yaml_* to change the dependencies as needed.
+. Edit *_Gopkg.toml_* to change the dependencies as needed.
 
-. Delete *_glide.lock_* and re-create the vendor directory by running `make vendor`.
-Glide will recognize that there is no lock file and recalculate the required dependencies.
+. Delete *_Gopkg.lock_* and re-create the vendor directory by running `make vendor`.
+Dep will recognize that there is no lock file and recalculate the required dependencies.
 
-. Check-in the updated *_glide.yaml_* and *_glide.lock_* files.
+. Check-in the updated *_Gopkg.toml_* and *_Gopkg.lock_* files.
 
 . Test that everything still compiles with the new lock file in place by running `make clean && make`.
 
 [TIP]
 ====
-The Glide cache located under *_~/.glide/cache_* can get corrupted in some cases.
-If you see Glide errors during `make vendor`, you can clear the Glide cache via `glide cc`.
+The Dep cache located under *_$GOPATH/pkg/dep_*.
+If you see any Dep errors during `make vendor`, you can remove local cached directory and try again.
```

Comment:
We wrote this tip for glide because we were getting an error during `glide install` but right now we are not sure we need for dep but I have it this as a placeholder and if this change then we will update accordingly.